apps/voting/src/shared/wallet-connection.tsx

```diff
@@ -125,7 +125,7 @@ function useWalletConnector(wc: WalletConnector): ConnectorContext {
                 );
 
                 if (temp === undefined) {
-                    return undefined;
+                    break outer;
```

Comment:
I don't understand this change.

Isn't the behaviour the same, since if this case is reached `conn` remains undefined.

Or am I reading the nesting wrong?